api/ocm/extensions/attrs/preferrelativeattr/attr.go

```diff
@@ -0,0 +1,66 @@
+package preferrelativeattr
+
+import (
+	"fmt"
+
+	"ocm.software/ocm/api/datacontext"
+	"ocm.software/ocm/api/utils/runtime"
+)
+
+const (
+	ATTR_KEY   = "github.com/mandelsoft/ocm/preferrelativeaccess"
```

Comment:
This attribute key should be changed.
```suggestion
	ATTR_KEY   = "ocm.software/ocm/oci/" + ATTR_SHORT
```